### PR TITLE
Enhancements

### PR DESCRIPTION
**Pull request by ricky1uphealth**
GF#21836
GF#21657
GF#21935
GF#21751
GF#21862

### CODE REVIEW

File: spec/index.md [hunk 1]
```diff
@@ -25,6 +25,11 @@ This use case exports all resources needed for the US Core Data for Interoperabi
 
 This use case exports all resources needed to convey a patient's healththcare financial history, including Patient, ExplanationOfBenefit, Coverage, and Claim. While FHIR profiles are still being developed and standardized, see [https://bluebutton.cms.gov/developers/#core-resources](https://bluebutton.cms.gov/developers/#core-resources) for a full-fledged example.
 
+### Additional Use cases
+* Terminology data  e.g. to allow export of all ValueSet from a terminology server.
+* Provider data to export information about an system's full Practitioner, Location, and Organization list
+* Use in training machine learning algorithms on historical data
```
Comment (jmandel) [inline spec/index.md]: Can you point me toward the GForge resolution where this came up? The other use cases are describing server functionality rather than what clients will do with data downstream. See this content in GF$21657.

Comment (ricky1uphealth) [reply spec/index.md]: The provider use case is added based on the comments you (https://github.com/HL7/bulk-data/pull/3#discussion_r295600205) and other made. The ML use case I added as another, but it wasn't requested by anyone during the balloting comments. I believe it's a valid use case but we can remove it if you disagree or if that's not following protocol

File: spec/export/index.md [hunk 2]
```diff
@@ -78,6 +84,24 @@ Export data from a FHIR server whether or not it is associated with a patient. T
 
   Specifies the format of the optional OperationOutcome response to the kick-off request. Currently, only application/fhir+json is supported.
```
Comment (jmandel) [inline spec/export/index.md]:
```suggestion
  Specifies the format of the optional OperationOutcome response to the kick-off request. Currently, only `application/fhir+json` is supported.
```

File: spec/export/index.md [hunk 3]
```diff
@@ -78,6 +84,24 @@ Export data from a FHIR server whether or not it is associated with a patient. T
...
 
+  **Example OperationOutcome**
```
Comment (jmandel) [inline spec/export/index.md]: Why is this here? It looks like an error that might be returned in response to fetching an exported file, but it's sitting in the "kickoff" portion of the spec.

File: spec/index.md (hunk 1 above)
Comment (jmandel) [reply spec/index.md]: There are many, many valid use cases -- but we need to implement the specific changes that were requested in the ballot and adjudicated/voted on by the FHIR Infrastructure Workgroup. We are not at liberty to craft new language without proper review.

Comment (jmandel) [reply spec/index.md]: There are many, many valid use cases -- but we need to implement the specific changes that were requested in the ballot and adjudicated/voted on by the FHIR Infrastructure Workgroup. We are not at liberty to craft new language without proper review.

Comment (ricky1uphealth) [reply spec/index.md]: Yup got it. Ive removed that ML example

**Comment by ricky1uphealth** [comment]
@jmandel Ive addressed all the changes requested. Ready for your review

File: spec/export/index.md (hunk 3 above)
Comment (jmandel) [reply spec/export/index.md]: @ricky1uphealth This issues still needs to be addressed

Comment (ricky1uphealth) [reply spec/export/index.md]: Gotcha. I moved it to the correct section. It's fixed here https://github.com/HL7/bulk-data/pull/7/files#diff-985cef1e2f9cd85436f27813aa6de763R206

**Comment by ricky1uphealth** [comment]
move `Even if some of the requested resources cannot successfully be exported, the overall export operation MAY still succeed. ` to below the operation outcome example and add `Note:` in front

File: spec/export/index.md (hunk 2 above)
Comment (ricky1uphealth) [reply spec/export/index.md]: @ricky1uphealth commit this change

Comment (ricky1uphealth) [reply spec/export/index.md]: this had already been fixed https://github.com/1uphealth/bulk-data/blame/enhancements/spec/export/index.md#L91

**Comment by ricky1uphealth** [comment]
@jmandel the requested changes have been addressed please review when you get a chance
